tests/dvslib/dvs_vlan.py

```diff
@@ -0,0 +1,94 @@
+from dvs_database import DVSDatabase
+
+class DVSVlan(object):
+    def __init__(self, adb, cdb, sdb, cntrdb, appdb):
+        self.asic_db = adb
+        self.config_db = cdb
+        self.state_db = sdb
+        self.counters_db = cntrdb
+        self.app_db = appdb
+
+    def create_vlan(self, vlanID):
+        vlanName = "Vlan{}".format(vlanID)
+        vlan_entry = {"vlanid": vlanID}
+        self.config_db.create_entry("VLAN", vlanName, vlan_entry)
+
+    def remove_vlan(self, vlanID):
+        vlanName = "Vlan{}".format(vlanID)
+        self.config_db.delete_entry("VLAN", vlanName)
+
+    def create_vlan_member(self, vlan, interface, tagging_mode="untagged"):
+        member = "Vlan{}|{}".format(vlan, interface)
+        if tagging_mode:
+            member_entry = {"tagging_mode": tagging_mode}
+        else:
+            member_entry = {"no_tag_mode": ""}
+
+        self.config_db.create_entry("VLAN_MEMBER", member, member_entry)
+
+    def remove_vlan_member(self, vlan, interface):
+        member = "Vlan{}|{}".format(vlan, interface)
+        self.config_db.delete_entry("VLAN_MEMBER", member)
+
+    def create_port_channel(self, lag_id, admin_status="up", mtu="1500"):
```

Comment:
Can you reuse all the port-channel related function like `create_port_channel, remove_port_channel, create_port_channel_member, remove_port_channel_member` from **`tests/dvslib/dvs_lag.py`**?

Reply:
yeah, seems these are copied by mistake here? looks same as in dvs_lag.py

Reply:
Thanks. These were part of this class, later I created dvs_lag.py, but forgot to delete them from here. Let me do it.